debian/base/include/etc/confluent/docker/mesos-setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [ -z $SKIP_MESOS_AUTO_SETUP ]; then
```

Comment:
Can this check for a positive condition (like ENABLE_MESOS) instead?

Reply:
Totally understand positives are preferred when coding. I'm making sure users who choose to skip this really knows what they are doing. They should be using the default setup unless they really really want to manually configure.